src/main/kotlin/fi/metatavu/timebank/api/controllers/PersonsController.kt

```diff
@@ -125,19 +106,19 @@ class PersonsController {
     suspend fun listPersons(active: Boolean? = true): List<ForecastPerson>? {
         val persons = getPersonsFromForecast()
 
-        persons.forEach { forecastPerson ->
-            val vacationAmounts = vacationUtils.getPersonsVacations(forecastPerson)
-            forecastPerson.unspentVacations = vacationAmounts.first
-            forecastPerson.spentVacations = vacationAmounts.second
-        }
-
         return if (active == false) {
             persons
         } else {
             filterPersons(persons)
         }
     }
 
+    suspend fun findPerson(personId: Int): ForecastPerson? {
+        val person = forecastService.findPerson(personId)
+
+        return person
+    }
```

Comment:
```suggestion
    suspend fun findPerson(personId: Int): ForecastPerson? {
        return forecastService.findPerson(personId)
    }
```

You can directly return with either as in the suggestion or following
`suspend fun findPerson(personId: Int): ForecastPerson? = forecastService.findPerson(personId)`